Dockerfile

```diff
@@ -34,6 +34,10 @@ RUN apt-get update \
     && sed 's/#.*//' /tmp/apt-run.txt | xargs apt-get install -y \
     && rm -rf /var/lib/apt/lists/*
 
+# Read configure from docker image folder
+COPY production /crop-mask/production
+COPY testing /crop-mask/testing
+
```

Comment:
Perhaps only include production? Testing is run on a small scale and it is probably more flexible not to have the assets static in the image.

Reply:
Don't be confused by the 'testing' terminology, the production models etc are stored in the testing folder as they are created using notebooks.

Reply:
@fivejjs I agree that we should be explicit with the files in this case (even thogh I like the simplicity of just copying one folder).  Lets just copy the `td`, `model`, and `aez` files into the docker image.  Then when we run the other models for the other regions, I'll just keep adding new file paths to the `Dockerfile`

Reply:
sure.

Reply:
Some of the  files got the timestamp
```
/crop-mask/testing/eastern_cropmask/results/gm_mads_two_seasons_ml_model_20210427.joblib
```
We add `/crop-mask/testing/eastern_cropmask/results/gm_mads_two_seasons_ml_model_*`